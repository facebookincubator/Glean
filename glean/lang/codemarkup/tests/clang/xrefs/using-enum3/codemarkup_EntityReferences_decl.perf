{ "@generated": null, "cxx1.Declarations.5": 8, "cxx1.TargetUses.5": 5 }