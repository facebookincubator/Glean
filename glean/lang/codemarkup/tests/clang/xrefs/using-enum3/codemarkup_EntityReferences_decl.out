[
  "@generated",
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "enum_": {
              "key": {
                "name": {
                  "key": {
                    "name": { "key": "E" },
                    "scope": {
                      "namespace_": { "key": { "name": { "key": "N1" } } }
                    }
                  }
                },
                "isScoped": false,
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 11,
                  "columnBegin": 1,
                  "lineEnd": 11,
                  "columnEnd": 18
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 307, "length": 1 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "enum_": {
              "key": {
                "name": {
                  "key": {
                    "name": { "key": "E" },
                    "scope": {
                      "namespace_": { "key": { "name": { "key": "N1" } } }
                    }
                  }
                },
                "isScoped": false,
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 11,
                  "columnBegin": 1,
                  "lineEnd": 11,
                  "columnEnd": 18
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 400, "length": 1 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "enum_": {
              "key": {
                "name": {
                  "key": {
                    "name": { "key": "E" },
                    "scope": {
                      "namespace_": { "key": { "name": { "key": "N1" } } }
                    }
                  }
                },
                "isScoped": false,
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 11,
                  "columnBegin": 1,
                  "lineEnd": 11,
                  "columnEnd": 18
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 526, "length": 1 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "enum_": {
              "key": {
                "name": {
                  "key": {
                    "name": { "key": "E" },
                    "scope": {
                      "namespace_": { "key": { "name": { "key": "N1" } } }
                    }
                  }
                },
                "isScoped": false,
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 11,
                  "columnBegin": 1,
                  "lineEnd": 11,
                  "columnEnd": 18
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 581, "length": 1 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "enum_": {
              "key": {
                "name": {
                  "key": {
                    "name": { "key": "E" },
                    "scope": {
                      "namespace_": { "key": { "name": { "key": "N1" } } }
                    }
                  }
                },
                "isScoped": false,
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 11,
                  "columnBegin": 1,
                  "lineEnd": 11,
                  "columnEnd": 18
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 670, "length": 1 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "enum_": {
              "key": {
                "name": {
                  "key": {
                    "name": { "key": "E" },
                    "scope": {
                      "namespace_": { "key": { "name": { "key": "N1" } } }
                    }
                  }
                },
                "isScoped": false,
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 11,
                  "columnBegin": 1,
                  "lineEnd": 11,
                  "columnEnd": 18
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 725, "length": 1 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "namespace_": {
              "key": {
                "name": { "key": { "name": { "key": "N1" } } },
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 9,
                  "columnBegin": 1,
                  "lineEnd": 13,
                  "columnEnd": 1
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 293, "length": 2 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "namespace_": {
              "key": {
                "name": { "key": { "name": { "key": "N1" } } },
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 9,
                  "columnBegin": 1,
                  "lineEnd": 13,
                  "columnEnd": 1
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 303, "length": 2 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "namespace_": {
              "key": {
                "name": { "key": { "name": { "key": "N1" } } },
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 9,
                  "columnBegin": 1,
                  "lineEnd": 13,
                  "columnEnd": 1
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 396, "length": 2 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "namespace_": {
              "key": {
                "name": { "key": { "name": { "key": "N1" } } },
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 9,
                  "columnBegin": 1,
                  "lineEnd": 13,
                  "columnEnd": 1
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 419, "length": 2 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "namespace_": {
              "key": {
                "name": { "key": { "name": { "key": "N1" } } },
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 9,
                  "columnBegin": 1,
                  "lineEnd": 13,
                  "columnEnd": 1
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 509, "length": 2 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "namespace_": {
              "key": {
                "name": { "key": { "name": { "key": "N1" } } },
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 9,
                  "columnBegin": 1,
                  "lineEnd": 13,
                  "columnEnd": 1
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 522, "length": 2 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "namespace_": {
              "key": {
                "name": { "key": { "name": { "key": "N1" } } },
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 9,
                  "columnBegin": 1,
                  "lineEnd": 13,
                  "columnEnd": 1
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 577, "length": 2 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "namespace_": {
              "key": {
                "name": { "key": { "name": { "key": "N1" } } },
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 9,
                  "columnBegin": 1,
                  "lineEnd": 13,
                  "columnEnd": 1
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 590, "length": 2 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "namespace_": {
              "key": {
                "name": { "key": { "name": { "key": "N1" } } },
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 9,
                  "columnBegin": 1,
                  "lineEnd": 13,
                  "columnEnd": 1
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 666, "length": 2 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "namespace_": {
              "key": {
                "name": { "key": { "name": { "key": "N1" } } },
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 9,
                  "columnBegin": 1,
                  "lineEnd": 13,
                  "columnEnd": 1
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 721, "length": 2 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "namespace_": {
              "key": {
                "name": { "key": { "name": { "key": "N21" } } },
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 15,
                  "columnBegin": 1,
                  "lineEnd": 24,
                  "columnEnd": 1
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 655, "length": 3 } }
    }
  },
  {
    "key": {
      "target": {
        "cxx": {
          "decl": {
            "namespace_": {
              "key": {
                "name": { "key": { "name": { "key": "N22" } } },
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 26,
                  "columnBegin": 1,
                  "lineEnd": 35,
                  "columnEnd": 1
                }
              }
            }
          }
        }
      },
      "file": { "key": "test.cpp" },
      "range": { "span": { "start": 744, "length": 3 } }
    }
  }
]